src/ft/mod.rs

```diff
@@ -347,6 +348,11 @@ impl FreeTypeRasterizer {
 
             let pixelsize_fixup_factor = pattern.pixelsizefixupfactor().next();
 
+            let bgr = match pattern.rgba().next().unwrap_or(fc::Rgba::Unknown) {
+                fc::Rgba::Bgr | fc::Rgba::Vbgr => true,
+                _ => false,
+            };
```

Comment:
Could we pass subpixel type around instead of bool flag, I don't like random bool flags, but on the other side other subpixel options won't be used?

What do you think @chrisduerr

Reply:
I completely agree. I also don't think it makes sense to match on an enum like that.